docs/latest/tutorial/tutorial-os-integration.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'OS integration'
+description: 'This guide will step you through the process of creating a barebones Hello World app in Electron, similar to electron/electron-quick-start.'
+slug: os-integration
+hide_title: false
+---
+
+:::info Tutorial parts
+This is part 4 of the Electron tutorial. The other parts are:
+
+- [Prerequisites][prerequisites]
+- [Scaffolding][scaffolding]
+- [Accessing Node.js APIs][node-access]
+- [Deeper integration with the OS][os-integration]
+- [Application distribution]
+- [Code signing]
+- [Updating your application][updates]
+
+:::
+
+There are many ways to integrate with the Operating System. In some cases it could be platform specific
+(like Apple's TouchBar), in others it could be through APIs to have a more integrated look and feel
+(like modifying the Title Bar), or using APIs that give you access to USB and Bluetooth devices.
+
+The following is a list of all the guides currently available. If there is still something that you do
+not know how to do, please take a look at the [API][app] as there is a chance it might be documented
+just there (and also open an issue requesting the guide!).
```

Comment:
> I think that repeating content is fine, isn't there a say such as "when you are sick of repeating yourself, that's when people start listening to you"? or something around those lines? We have had this content for a longtime yet people are having trouble finding it.

My only issue with repetition in writing is that it makes it easier for docs to go stale. I think making it easier for people to find the single place where the information is stored would be the way to go.

IMO the best way would be to go about doing this would be a few different steps:

1) Split off the "How-To" category in the sidebar into a "Desktop Environment Integration" category and... one or more other ones (TBD).

1) The "Desktop Environment Integration" section should have an "Overview" doc that contains this content.

1) Rename the `Managing your window's lifecycle` to `Desktop environment integration` or `Basic OS integration` because making the window lifecycle match OS-specific behaviour is part of that broader category, and add a note redirecting to the Overview doc from the last point.

I think a re-org like this would make more sense because you can find info in the same spots, but we're repeating a bit less.

Reply:
> My only issue with repetition in writing is that it makes it easier for docs to go stale. 

Yes, we need to do something about all the stale ones we have already 😔

I like your proposal!

> Split off the "How-To" category in the sidebar into a "Desktop Environment Integration" category and... one or more other ones (TBD).

What about:

- OS integration (thinking about the space we have in the sidebar)
- How to (just keep the rest there)

Reply:
> What about:

> OS integration (thinking about the space we have in the sidebar)
> How to (just keep the rest there)

I like that!

Reply:
@erickzhao do we want to link to OS Integration or Examples from the sidebar?